packages/sdk/src/types/internal/context.ts

```diff
@@ -103,13 +103,15 @@ export class InternalContext {
 		actor?: Partial<ActorLike>
 	}): Actor {
 		const payload = {
-			type: 'create-from-prefab',
-			prefabId: options.prefabId,
+			...options,
 			actor: {
-				...options.actor,
-				id: UUID()
-			}
+				...options.actor
+			},
+			type: 'create-from-prefab'
 		} as Payloads.CreateFromPrefab;
+		if (!payload.actor.id) {
+			payload.actor.id = UUID();
+		}

```

Comment:
I don't think this is safe. What if I want to use an existing actor as a template for a new one?